packages/components/src/color-picker/style.scss

```diff
@@ -35,7 +35,7 @@
 	position: relative;
 }
 .components-color-picker__body {
-	padding: 16px 16px 12px;
+	padding: 16px 0 12px;
```

Comment:
While we're here, can you use grid units? Essentially the following instead:

`padding: $grid-size-large 0 #{ $grid-size-small * 3 };`

Reply:
I've added this to the latest pull